geomesa-features/geomesa-feature-kryo/src/main/scala/org/locationtech/geomesa/features/kryo/ProjectingKryoFeatureDeserializer.scala

```diff
@@ -48,17 +51,26 @@ class ProjectingKryoFeatureDeserializer(original: SimpleFeatureType,
     }
   }
 
-  override def deserialize(bytes: Array[Byte]): SimpleFeature = deserialize(bytes, 0, bytes.length)
+  override def deserialize(bytes: Array[Byte]): SimpleFeature = deserialize("", bytes, 0, bytes.length)
+
+  override def deserialize(id: String, bytes: Array[Byte]): SimpleFeature = deserialize(id, bytes, 0, bytes.length)
+
+  override def deserialize(in: InputStream): SimpleFeature = throw new NotImplementedError
+
+  override def deserialize(id: String, in: InputStream): SimpleFeature = throw new NotImplementedError
+
+  override def deserialize(bytes: Array[Byte], offset: Int, length: Int): SimpleFeature =
```

Comment:
Is it the case that some of these implementations are obvious defaults that could be moved back to the trait?

Reply:
yeah, good idea